Resources/translations/FOSUserBundle.da.yml

```diff
@@ -40,11 +46,17 @@ registration:
 
             For at færdiggøre aktiveringen af din konto, besøg venligst %confirmationUrl%
 
+            Dette link kan kun bruges én gang til at validere din konto.
+
             Venlig hilsen,
             Holdet bag siden.
 
 resetting:
-    check_email: 'En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.'
+    check_email: |
+    En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.
```

Comment:
Sorry. I’m new to Git/Github and got some files mixed up. I have fixed the syntax error but I’m not sure what to do now.